### ISSUE TITLE
Java client issue "No appropriate protocol (protocol is disabled or cipher suites are inappropriate)"

### ISSUE THREAD

**Issue by fnJeff** [TigerVNC/tigervnc#851]
I've been using the java client 1.9.0 version for quite some time without any issues.  I believe this problem is related to the newest java version.  My connection was active and then it just dropped and would not reconnect.  At some point I'm thinking that java was updated underneath it, but I have no idea if that is even possible.  What I know is that it was connected, then disconnected and would not reconnect with the following popup.

![image](https://user-images.githubusercontent.com/31246517/61352928-d9798700-a86e-11e9-98e2-9fa9f96238b6.png)

My work PC currently has this version of java installed, this is the one that no longer works.
![image](https://user-images.githubusercontent.com/31246517/61353022-09288f00-a86f-11e9-9bc3-fbf46f8d3bc2.png)

I tried connecting from my home PC and it worked normally, but has what I think is the previous java version still.
![image](https://user-images.githubusercontent.com/31246517/61353061-23fb0380-a86f-11e9-85b2-e179b3742502.png)

I can connect fine from my work PC using the windows client, so I'm fairly certain it's not a server issue.  Both PCs are windows 10.

Hope you can help figure this out.  The windows client does not support remote scaling like the java client does, and that's the main reason I use the java client.  My company IT only officially supports the windows client, so they won't help me.  I'm hoping it's an easy thing to find and fix.

**Comment by fnJeff** [comment]
Quick update.  I just did some more searching and someone replied to different but related topic a few weeks after I created this issue.

https://stackoverflow.com/questions/38205947/sslhandshakeexception-no-appropriate-protocol

It seems something changed in the java.security file.  When I comment out the line mentioned in the newest answer I can connect again.

I tried commenting out each protocol 1 by 1 to try and identify which one being disabled caused the issue, but it failed each time.  It seems only commenting out the entire line allows it to work again.
